### PR TITLE
feature: add support for custom pgns

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: n2kMapper.js [hunk 1]
```diff
@@ -16,8 +16,26 @@ Object.assign(n2kMappings, require('./actisense'))
...
 function N2kMapper (options) {
-  this.state = {}
+  this.state = {  }
   this.unknownPGNs = {}
+  this.customPgns = {}
+  this.options = options || {}
+  
+  if ( this.options.onPropertyValues ) {
+    this.options.onPropertyValues('pgn-to-signalk', values => {
+      values.filter(v => v != null).forEach(pv => {
```
Comment (tkurki) [inline n2kMapper.js:26]: This is to discard the first value `undefined`, right? I think the test should be `!== undefined`, as this is not about null and null comparisons are needlessly tricky:

```
undefined != null
false
undefined !== null
true
```

**Review by tkurki — COMMENTED** [review]
Some documentation in README?

Is `allowCustomPGNOverride` already set up in the server?

File: n2kMapper.js (hunk 1 above)
Comment (sbender9) [reply n2kMapper.js]: fixed

**Comment by sbender9** [comment]
> Some documentation in README?
> 
> Is `allowCustomPGNOverride` already set up in the server?

I wasn't going to add this to the server. It's going to be a very rare use case.

**Pull request merged**